### ISSUE TITLE
DataTable - `groupBy` expand/collapse control doesn't receive focus indicator

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7336]
### Expected Behavior
The expand/collapse control should have a focus indicator around it when it receives keyboard focus.

### Actual Behavior
Currently it does not have a focus indicator.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/visualizations-datatable-controlled-grouped--controlled-grouped-data-table

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by britt6612** [comment]
also fails WCAG 2.4.7 Focus Visible
